tenant/pkg/webhook/default_server/tenantnamespace/validating/tenantnamespace_create_update_handler.go

```diff
@@ -61,21 +87,38 @@ func (h *TenantNamespaceCreateUpdateHandler) Handle(ctx context.Context, req typ
 	if err != nil {
 		return admission.ErrorResponse(http.StatusBadRequest, err)
 	}
-
-	allowed, reason, err := h.validatingTenantNamespaceFn(ctx, obj)
-	if err != nil {
-		return admission.ErrorResponse(http.StatusInternalServerError, err)
+	switch req.AdmissionRequest.Operation {
+	case admissionv1beta1.Create:
+		// Fetch tenant list
+		tenantList := &tenancyv1alpha1.TenantList{}
+		err := h.Client.List(ctx, &client.ListOptions{}, tenantList)
+		if err != nil {
+			return admission.ErrorResponse(http.StatusInternalServerError, fmt.Errorf("cannot validate tenantnamespace CR because client cannot get tenant list"))
```

Comment:
Nit: could use errors.New() since you're not formatting anything.

Reply:
I will change it in next change.